pkg/proxy/ipvs/proxier.go

```diff
@@ -675,6 +679,10 @@ func (handle *LinuxKernelHandler) GetKernelVersion() (string, error) {
 	return strings.TrimSpace(string(fileContent)), nil
 }
 
+// Store kernelVersion so we can run handler once and use the value to check if certain sysctls
+// should not be run on older kernels
+var kernelVersion *version.Version
```

Comment:
not a fan of this global var, any way we can refactor to pass in kernelVersion as a parameter to NewProxier?

Reply:
Agreed, it makes the code harder to understand
I like the idea of adding it to NewProxier (we have everything we need in `newProxyServer`)

Reply:
I will try to work on that. The reason I settled on the variable was because it felt like all sysctl stuff should probably be in it's own function since NewProxy is not the only thing that touches sysctls, but the new function is probably a larger TODO than this bug fix.